sdk/evaluation/azure-ai-evaluation/tests/conftest.py

```diff
@@ -177,6 +177,34 @@ async def combined_call(*args, **kwargs):
     AsyncioRequestsTransport.send = original_transport_func
 
 
+@pytest.fixture
+def simple_conversation():
+    return {
+        "messages" : [
+            {
+                "content": "What is the capital of France?",
+                "role": "user",
+                "context": "Customer wants to know the capital of France"
+            },
+            {
+                "content": "Paris",
+                "role": "assistant",
+                "context": "Paris is the capital of France"
+            },
+            {
+                "content": "What is the capital of Hawaii?",
+                "role": "user",
+                "context": "Customer wants to know the capital of Hawaii"
+            },
+            {
+                "content": "Honolulu",
+                "role": "assistant",
+                "context": "Honolulu is the capital of Hawaii"
+            }
+        ],
+        "context": "Global context"
```

Comment:
Curious what's "global context"? who will provide "global context"? Asking is because I want to know whether it can be covered by system message? for example:

            {
                "content": "You are an AI assistant.",
                "role": "system",
                "context": "Customer wants to know the capital of France"
            },

Reply:
The expected format of a conversation is a dictionary. Both the conversation turns AND the conversation as a whole can potentially have 'context' as a key with a string value. Nagkumar is currently ensuring that simulators abide by this expectation here, even though the global context will often be empty: https://github.com/Azure/azure-sdk-for-python/pull/37575/files#diff-178ca75cf46816f714886415ae98fe2416f4d216cc470b469bcf15770e86bc4fR236